metrics/prometheus.go

```diff
@@ -415,6 +415,32 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 			},
 		}...)
 	}
+	if includedMetrics.Has(container.PidsMetrics) {
```

Comment:
We already have container_processes.  How does this differ from that metric?

Reply:
`ProcessCount`  just collect processes num, and it is not equals to threads num in container.

Reply:
`ProcessCount`  just collect processes num, and it is not equals to threads num in container.

Reply:
I think I would expect these to match.  `ProcessCount` counts the number of pids listed in the `cgroups.proc` file in the `cpu` hierarchy where the new stat just reads the `pids.current` in the `pids` hierarchy.

Reply:
pids.current equals to processes + threads num，its value is like pstree-l entries num

Reply:
This is different from threads, right?  Or if this is a threads metric, lets name it that way

Reply:
@dashpole it's threads metrics, I will rename it by prefix like `Threads(Max|Current)`